src/sass/_common/common.scss

```diff
@@ -947,33 +947,142 @@ label + input {
 }
 
 #token_form {
-    @media screen and (max-width: 480px) {
+    fieldset {
+        padding: 14px;
+    }
+    fieldset > legend {
+        margin-left: 20px;
+    }
+    .title-row {
+        margin: 20px 2px 5px;
+
+        label {
+            white-space: nowrap;
+        }
+        input {
+            width: 205px;
+            height: 22px;
+            max-width: 100%;
+        }
+    }
+    .scopes-title {
+        padding: 0;
+        margin: 0;
+
+        label {
+            margin: 0 0 0 13px;
+            text-align: left;
+        }
+    }
+    .scope-types {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
```

Comment:
You can use grid here to more easily divide elements in an equal size into 2 columns. Then you don't have to hardcode width and height on your scope items (see comment I made further below).

You could do: 
`
display: grid;
`
`grid-template-columns: 1fr 1fr;`